invisible-wallet/typescript/src/invisible_wallet_examples.ts

```diff
@@ -1,169 +1,170 @@
-import { ExecuteTransactionRequestType, SuiClient, SuiTransactionBlockResponse } from "@mysten/sui.js/client";
+// 1. Import everything we'll need for the rest of the tutorial
+import { 
+  WalletClient, 
+  KeyClient, 
+  createSuiClient, 
+  GasStationClient,
+  buildGaslessTransactionBytes, 
+  ShinamiWalletSigner
+} from "@shinami/clients";
 import { TransactionBlock } from "@mysten/sui.js/transactions";
-import { rpcClient } from "typed-rpc";
+import { verifyPersonalMessage } from '@mysten/sui.js/verify';
+
+// 2. Copy your access key value
+const ALL_SERVICES_TESTNET_ACCESS_KEY = "{{allServicesTestnetAccessKey}}";
+
+// 3. Set up a wallet id and an associated sercret
+const WALLET_ONE_ID = "{{walletOneId}}";
+const WALLET_ONE_SECRET = "{{walletOneSecret}}";
+
+
+// 4. Instantiate your Shinami clients
+const keyClient = new KeyClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const walletClient = new WalletClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const nodeClient = createSuiClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+
+// 5. Set up a variable we'll use in the examples below
+let senderSignature = null;
+
+// 6. Generate a signer for our invisible wallet and try to create it
+const signer = new ShinamiWalletSigner(
+  WALLET_ONE_ID,
+  walletClient,
+  WALLET_ONE_SECRET,
+  keyClient
+);
+// Returns the Sui address if it creates a wallet, otherwise undefined if the wallet exists
+let addressOrUndefined = await signer.tryCreate();
+
+// 7. If undefined, we can look up the  address with the `gatAddress` method
+const WALLET_ONE_SUI_ADDRESS = addressOrUndefined ? addressOrUndefined : await signer.getAddress();
```

Comment:
You can directly use `signer.getAddress(true)` which saves you the trouble of calling `signer.tryCreate()`.

Reply:
updated